shared/media-accelerator/develop/_ensure-channel-quality.mdx

```diff
@@ -0,0 +1,101 @@
+
+## Understand the tech
+
+<Vpd k="SDK"/> provides the following features to deal with channel quality issues:
+
+- **Choose an audio codec to optimize bit rate and quality**
+
+  In audio and video streaming, the choice of an encoding algorithm affects both quality and bit rate. Your goal is to lower the required bit rate while maintaining quality at the desired level. <Vpd k="SDK" /> offers the following built-in audio codecs to choose from:
+
+  - Opus
+  - G722
+  - G711A (PCMU) 
+  - G711U (PCMU) 
+
+  You specify an audio codec when you join a channel. You also set the sampling rate and the number of channels at this time, depending on your audio quality requirements. If your <Vpl k="CLIENT"/> needs a custom encoding algorithm, you disable use of a built-in codec and implement your own encoder. 
+
+- **Adjust sending bit rate in real time**
+
+  In order to optimize data transmission and avoid network congestion, <Vpd k="SDK"/> suggests adjusting the sending bit rate in real-time according to changes in network conditions.
+
+  You configure BandWidth Estimation (BWE) before joining a channel to set the minimum, maximum, and starting bit rate values according to the actual bandwidth and bit rate needs. When the network bandwidth changes, <Vpd k="SDK"/> triggers an event to prompt your <Vpl k="CLIENT"/> to adjust the sending bit rate in real time. The bit rate returned by the callback is the maximum recommended encoding bit rate of the video encoder.
+
+- **Change audio and video streaming status**
+
+  Network traffic can be reduced by suspending data transmission when audio or video feed is not required by the receiver. After a user successfully connects to a channel and starts audio and video streaming, they can suspend sending streams to a specific connection or to all connections to flexibly manage the transmission status of audio and video streams. Similarly, the user may suspend receiving a specific or all streams as required.
+
+  When a user changes the transmission status of the local audio or video stream, the <Vpd k="SDK"/> triggers a corresponding callback to prompt the remote user to suspend or resume sending their audio or video stream.
+
+- **Log files** 
+
+  <Vpd k="SDK"/> provides configuration options that you use to customize the location, content and size of log files containing key data of <Vpd k="SDK"/> operation. When you set up logging, <Vpd k="SDK"/> writes information messages, warnings, and errors regarding activities such as initialization, configuration, connection and disconnection to log files. Log files are useful in detecting and resolving channel quality issues.
+
+The following figure shows the workflow you need to implement to ensure channel quality in your <Vpl k="CLIENT"/>:
+
+![Ensure Channel Quality](/images/media-accelerator/media-accelerator-channel-quality.svg)
```

Comment:
Why are you saying service here when we are talking about an app?

Reply:
Changed service to engine